os_migrate/localhost_inventory.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
Well it's already fixed everywhere and it should be something standard for YAML in the beginning of each file, I dont think it hurts having it as it's already done :)

Reply:
Yea we'll just have to keep doing it in every new file too :grin:

Reply:
Discussed on IRC, we don't have 100% the same feeling about this but it's a small issue that is easy to reverse if we decide that later. We should merge lints ASAP so that we can continue our work with lints enabled and removed risk of conflicts with the lint patches. So merging.